spec/models/notification_spec.rb

```diff
@@ -8,6 +8,16 @@
   let(:article)         { create(:article, user_id: user.id, page_views_count: 4000, positive_reactions_count: 70) }
   let(:follow_instance) { user.follow(user2) }
 
+  describe "validations" do
```

Comment:
this test is fine but shoulda-matcher gem would make this a lot easier
```
 it { is_expected.to validate_uniqueness_of(:user_id).scoped_to(%i[organization_id notifiable_id notifiable_type action]) }
```

Reply:
I'm aware of the shoulda matchers, but they seem kind of less explicit to me 🤷🏻
Though, in this case, I agree that using shoulda matcher will be enough.